### ISSUE TITLE
SendRawTransaction blocks all other rpc commands until finished

### ISSUE THREAD

**Issue by cryptodev81** [vergecurrency/VERGE#808]
## Context

We are using the raw transactions functionality to sign and submit transactions. Whenever we are calling the sendrawtransaction ALL RPC functionality are giving timeouts until the sendrawtransaction call is completed (which takes about 3 minutes)

## Prerequisites

We are running on the 4.0.2a version which is, at the moment, the latest version.

## User Story:

* "I'm submitting a ... "
  - [x] bug report

We have Verge client running with RPC server setup. We are talking to this RPC to create, sign and submit raw transactions. Whenever we call the sendrawtransaction all other RPC commands are being blocked and causing timeout issues on our end. 

## Expected Behavior

Whenever we call the sendrawtransaction command we expect that other commands still work (ie, getbalance, getlatestblock or even another sendrawtransaction)

## Current Behavior

Whenever we call the sendrawtransaction command no other rpc commands are executing anymore.

## Possible Solution

Could we handle the sendrawtransaction asynchronously?

## Steps to Reproduce (for bugs)

Basically just send a raw transaction ... Not sure if this needs more 

## Your Environment

* Verge Release: 4.0.2a
* CPU: 4 Core on Azure VM (b4ms)
* RAM: 16Gb
* OS:
    - [x] I'm using Windows

**Comment by jrsikors** [comment]
Unfortunately I have to give you the same generic answer as most people...

We are in the home stretch of rebasing to Bitcoin v0.17 , which should resolve these types of issues.  I don't think Justin is going to look into resolving any of those types of things on 4.0.x

Out of curiosity, you said:

> no other rpc commands are executing anymore.

Indicating they did at one point in a previous version of Verge?  Or was that an accidental "anymore".

**Comment by Positivism** [comment]
> no other rpc commands are executing anymore
@jrsikors After executing sendrawtransaction they don't execute

**Comment by marpme** [comment]
@cryptodev81 Yep you're correct RPC calls sharing the same thread handle - it's single threaded. Everything else would make the process even more complex. 
Not sure tho, how complex your transactions are, but it would be interesting to know.

I would also suggest waiting for the new code rebase to be released.

**Comment by Positivism** [comment]
@oliverw Soon.

**Comment by jrsikors** [comment]
We are closing ALL issues to get a clean slate after the major codebase update.  Please re-submit issue with details if this is still an issue moving forward.
